doc/tutorials/run-application-with-data-from-API.md

```diff
@@ -0,0 +1,49 @@
+# Run Oscar application with sample data fetched from external API
+
+With the source code we provide predefined configuration file for fetching data from external API. For more info about this functionality check [URL loader service description](../services/url-loader.md).
+
+## What kind of data and from where?
+
+As most of the team members are from Slovakia, we chose to fetch a sample data related to Slovakia. COVID-19 API we used is created through automation platform [APIFY](https://apify.com/covid-19). All COVID-19 APIs there are open-source and free to use without any limitations. Data source for Slovakia COVID-19 API is [Coronavirus (COVID-19) in the Slovak Republic in numbers page](https://korona.gov.sk/en/coronavirus-covid-19-in-the-slovak-republic-in-numbers/). Latest data for one day in JSON format are available at this [URL](https://api.apify.com/v2/key-value-stores/GlTLAdXAuOz6bLAIO/records/LATEST?disableRedirect=true).
+
+## Configuration
+
+URL loader configuration file and all necessary environment variables are set by default. You can just run the scripts as in following section. You can find further description of configuration file properties [here](../data-structures/config-files.md#url-loader-config).
+
+./url-loader/sources/ExampleDailyUrls.yml
+
+```- name: APIFY Covid19 statistics
+  url: https://api.apify.com/v2/key-value-stores/GlTLAdXAuOz6bLAIO/records/
+  data:
+    - query: LATEST?disableRedirect=true
+      name: "Daily Covid19 cases in Slovakia"
+      filename: slovakia_daily_cases_{{date}}.json
+      foldername: slovakia_daily_cases
+```
+
+## Run Oscar application
+
+If this is your first attempt to run the application, please consult [Getting started](../getting-started/run-application.md) first. There are some necessary prerequisities.
+Then you need to run these three scripts:
+
+`./start.sh` will start the api, frontend and database. Once you can see the frontend, you can go to next script.
```

Comment:
First two scripts should be in the Getting started part, are they not?

Reply:
Yeah you are right, I will remove it. Should I remove it also from run-application-with-sample-data.md? There is something similar.

Reply:
Process described in this tutorial is not part of starting the application, it should be already running. So first of all, I would call this section:
## Run url-loader service
Before you start the service, please make sure, that Oscar application is running on your machine. If you use config file provided in this tutorial, check also that Sample data are loaded correctly. 
// continue with just ./runurlload.sh script